app/views/tag/blog.html.erb

```diff
@@ -74,7 +74,7 @@
         <hr />
       <% end %>
 
-    <div class="text-center"><%= will_paginate @notes, :renderer => BootstrapPagination::Rails unless @unpaginated %></div>
+    <%= will_paginate @notes, :renderer => BootstrapPagination::Rails unless @unpaginated %>
```

Comment:
https://github.com/publiclab/plots2/pull/3770/commits/fad97d71a0fde26a790541833e42142a5e61d434    this commit was right, please make changes to according to it
:)